src/lib/requests/mojaloopRequests.js

```diff
@@ -22,26 +22,32 @@ const { ResponseType } = require('./common');
 class MojaloopRequests extends BaseRequests {
     constructor(config) {
         super(config);
+
+        this._config = config;
     }
 
     /**
      * Executes a GET /parties request for the specified identifier type and identifier
      *
      * @returns {object} - JSON response body if one was received
      */
-    async getParties(idType, idValue, idSubValue, destFspId) {
+    async getParties(idType, idValue, idSubValue, destFspId, headers) {
         const url = `parties/${idType}/${idValue}`
             + (idSubValue ? `/${idSubValue}` : '');
-        return this._get(url, 'parties', destFspId);
+        
+        return headers
```

Comment:
Why conditional here and not the others?
Do we need this conditional?

Reply:
Good catch